src/spark/utils.py

```diff
@@ -59,6 +63,10 @@ def _get_delta_lake_conf(
         "spark.hadoop.fs.s3a.path.style.access": "true",
         "spark.hadoop.fs.s3a.impl": "org.apache.hadoop.fs.s3a.S3AFileSystem",
         "spark.sql.catalogImplementation": "hive",
+        # SparkMonitor extension configuration
+        # https://github.com/swan-cern/sparkmonitor?tab=readme-ov-file#setting-up-the-extension
+        "spark.extraListeners": "sparkmonitor.listener.JupyterSparkMonitorListener",
+        "spark.driver.extraClassPath": f"{site_packages_path}/sparkmonitor/listener_{SCALA_VER}.jar",
```

Comment:
This jar is built in? I don't see any gradle changes

Reply:
it comes with pip sparkmonitor.

Reply:
:+1: